velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -629,6 +634,27 @@ VectorPtr VectorFuzzer::fuzzComplex(const TypePtr& type, vector_size_t size) {
   return nullptr; // no-op.
 }
 
+VectorPtr VectorFuzzer::fuzzFlatOpaque(
+    const TypePtr& type,
+    vector_size_t size) {
+  auto vector = BaseVector::create(type, size, pool_);
+  using TFlat = typename KindToFlatVector<TypeKind::OPAQUE>::type;
+
+  auto& opaqueType = type->asOpaque();
+  auto flatVector = vector->as<TFlat>();
+  auto& opaqueTypeGenerator = opaqueTypeGenerators_[opaqueType.typeIndex()];
```

Comment:
nit: should we check first if a generator exists and return an error if it doesnt?